kernel/src/engine/mod.rs

```diff
@@ -0,0 +1,24 @@
+//! Provides engine implementation that implement the required traits. These engines can optionally
```

Comment:
This file is untouched, logic wise, from the old `client/mod.rs`. But with a change in the mod comment plus the feature flag name changes, github isn't smart enough to realize what happened and thinks it's a new file and that the old one was removed.